tests/test_doctest.py

```diff
@@ -58,4 +90,5 @@ class BadWeatherException(Exception):
         },
         optionflags=doctest.ELLIPSIS,
     )
-    sys.exit(results.failed)
+
+    sys.exit(results1.failed + results2.failed)
```

Comment:
This may return error code 2, should we make it so it's always 1 ?
```suggestion
    sys.exit(results1.failed | results2.failed)
```

Reply:
`failed` property actually returns the failed test count, but you are right. It's probably better to just always return 1 if tests fail.

Reply:
Fixed: https://github.com/flexmock/flexmock/pull/108/commits/dd1c121047b0d37aa622d946379360c50c0579d0